### PR TITLE
Change data model -> Add first half image url for domain model in popular movies

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: app/src/main/java/com/example/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class PopularMoviesDataRepository @Inject constructor(
                     movieRemote.name,
                     movieRemote.rating,
                     movieRemote.releaseDate,
-                    movieRemote.imageUrl
+                    "https://image.tmdb.org/t/p/w500" + movieRemote.imagePath
```
Comment (BorysChajdas) [inline app/src/main/java/com/example/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt]: Extract `"https://image.tmdb.org/t/p/w500"` as a some well-named const in repo

**Review by BorysChajdas — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/src/main/java/com/example/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt]: I added something like that `const val BASE_URL_FOR_IMAGE = "https://image.tmdb.org/t/p/w500"`

File: app/src/main/java/com/example/benchproject/data/common/constants/Constants.kt [hunk 2]
```diff
@@ -4,6 +4,7 @@ class Constants {
 
     companion object {
         const val BASE_URL = "https://api.themoviedb.org/3/movie/"
+        const val BASE_URL_FOR_IMAGE = "https://image.tmdb.org/t/p/w500"
```
Comment (BorysChajdas) [inline app/src/main/java/com/example/benchproject/data/common/constants/Constants.kt]: No, it should not be in some common file, it is not a good practice to hold all consts in one common place. Place it in the repository, as close as it's possible to the usage.

**Review by BorysChajdas — CHANGES_REQUESTED** [review]
Correct const

File: app/src/main/java/com/example/benchproject/data/common/constants/Constants.kt (hunk 2 above)
Comment (Nataniel-Antosik) [reply app/src/main/java/com/example/benchproject/data/common/constants/Constants.kt]: Ok I added companion object to the repo

**Review by BorysChajdas — APPROVED** [review]

**Pull request merged**